### PR TITLE
refactor: add constraints for yml of obo tab template

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

**Review by KennethBWSong — DISMISSED** [review]

File: templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl [hunk 1]
```diff
@@ -55,7 +56,7 @@ provision:
       # Relative path to this file. Environment variables in manifest will
       # be replaced before apply to AAD app
       manifestPath: ./aad.manifest.json
-      outputFilePath: ./build/aad.manifest.${{TEAMSFX_ENV}}.json
+      outputFilePath : ./build/aad.manifest.${{TEAMSFX_ENV}}.json
```
Comment (huimiu) [inline templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl:59]: There's an extra space here.

File: templates/ts/sso-tab-with-obo-flow/teamsapp.local.yml.tpl [hunk 2]
```diff
@@ -55,7 +56,7 @@ provision:
       # Relative path to this file. Environment variables in manifest will
       # be replaced before apply to AAD app
       manifestPath: ./aad.manifest.json
-      outputFilePath: ./build/aad.manifest.${{TEAMSFX_ENV}}.json
+      outputFilePath : ./build/aad.manifest.${{TEAMSFX_ENV}}.json
```
Comment (huimiu) [inline templates/ts/sso-tab-with-obo-flow/teamsapp.local.yml.tpl:59]: Same here.

File: templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl (hunk 1 above)
Comment (huimiu) [reply templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl:59]: There might be an extra space in the template for the `aadAppUpdate` action.

Comment (hund030) [reply templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl]: Updated.

**Review by huimiu — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**